src/FluidBufNNDSVD/CMakeLists.txt

```diff
@@ -19,3 +19,7 @@ add_library(
 )
 
 include(${CMAKE_CURRENT_LIST_DIR}/../../scripts/target_post.cmake)
+
+if(WIN32)
+  target_compile_options(${PLUGIN} PUBLIC /bigobj)   
```

Comment:
This doesn't feel like it should be on this PR? It's a general condition of compiling this object on Windows, no?

Reply:
Not sure to be honest. An error was being thrown:

https://github.com/jamesb93/flucoma-sc/runs/4122267956?check_suite_focus=true

and this fixes it...

Reply:
I remember this being a thing when this object first appeared, then it stopped being a problem but seems to have started again. I've love to get to the bottom of what causes it for this particular client (the implication would be that it's exposing lots and lots of symbols), but for the time being it should probably be its own commit rather than get squashed in with all the CI stuff: if it's a problem on the CI windows build, it'll be a problem on all the windows builds.

Reply:
> 

In addition I only ever have seen it for SC. Don't know if you've seen it for other environments?

Reply:
Okay I've split things up, so those changes to the client are reverted and have been extracted to a new PR:

https://github.com/flucoma/flucoma-sc/pull/41

This PR now depends on that being pulled in prior @weefuzzy

Reply:
> I only ever have seen it for SC. Don't know if you've seen it for other environments?

I think everything needs it for libmanipulation, and everything used to need it for NNDSVD. I'll get to the bottom of why one day, promise. 

Thanks for sorting